test/docs/variable_mapping_test.rb

```diff
@@ -151,10 +151,10 @@ module E
         module Memo; end
         #:no-input
         class Memo::Create < Trailblazer::Activity::Path
-          step :authorize, output: :authorize_output
+          step :authorize, output: :authorize_output, output_with_outer_ctx: true
```

Comment:
Could you add that as a separate test, please? From the diff it looks as if we're changing an existing test here to use an additional feature.